apps/emqx_bridge_snowflake/src/emqx_bridge_snowflake_lib.erl

```diff
@@ -93,7 +94,9 @@ jwt_config(ResId, Params) ->
         sub => Sub,
         aud => <<"unused">>,
         kid => <<"unused">>,
-        alg => <<"RS256">>
+        alg => <<"RS256">>,
+        table => emqx_connector_jwt,
+        grace_period => 30000
```

Comment:
1) Maybe there's no need to do this: default is 5 s, and this one does not involve HTTP requests.
2) Grace period is in [seconds](https://github.com/emqx/emqx/blob/b376c7696567b65aad65b804a6c8bb3284c8b8aa/apps/emqx_connector_jwt/src/emqx_connector_jwt.erl#L147).